### PR TITLE
Fix ESLint part 1

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
First part of actually fixing ESLint errors.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -187,7 +187,6 @@
   "lint-staged": {
     "*": "prettier -uw --cache",
     "*.{ts,tsx}": [
-      "eslint --cache --fix",
```
Comment (VIKTORVAV99) [inline web/package.json:190]: This prevented me from pushing the code until all ESlint errors are fixed. Will add it back once they are fixed.

File: web/.prettierignore [hunk 2]
```diff
@@ -9,6 +9,7 @@ dist/
 config/
 public/images/
 .husky/_
+pnpm-lock.yaml
```
Comment (VIKTORVAV99) [inline web/.prettierignore:12]: Not ESLint related but figured I might as well include it.

File: web/scripts/generate-index-files.ts [hunk 3]
```diff
@@ -13,13 +13,13 @@ const allPaths = [...generalPaths, ...zonePaths];
 let filesGenerated = 0;
```
Comment (madsnedergaard) [inline web/scripts/generate-index-files.ts:13]: This file is deleted :)

**Review by madsnedergaard — APPROVED** [review]
LGTM

File: web/scripts/generate-index-files.ts (hunk 3 above)
Comment (VIKTORVAV99) [reply web/scripts/generate-index-files.ts:13]: I'll update the PR to reflect the current state 👍🏼

**Pull request merged**